src/App.tsx

```diff
@@ -0,0 +1,20 @@
+import { FC } from 'react';
+import { Header } from './components/Header';
+import { Routes, Route } from 'react-router-dom';
+import { Home } from './pages/Home';
+import { Quiz } from './pages/Quiz';
+import { QuizReport } from './pages/QuizReport';
+
+export const App: FC = () => {
```

Comment:
is there any particular reason to use this?

Reply:
As far as I know, the return value is valid JSX.